onnx/utils.py

```diff
@@ -10,24 +10,11 @@
 
 import onnx.checker
 import onnx.helper
-import onnx.optimizer
 import onnx.shape_inference
 
 from onnx import ModelProto, NodeProto, TensorProto, ValueInfoProto
 
 
-def polish_model(model):  # type: (ModelProto) -> ModelProto
```

Comment:
you can keep this method just remove optimize and the checker call after optimize...

Reply:
If so this method will only run strip_doc_string and shape_inference. It makes this method not so useful. What's your opinion? Thanks!